test/blob_test.js

```diff
@@ -0,0 +1,67 @@
+suite.only("Azure Blob", function() {
+  var azure   = require('../');
+  var assert  = require('assert');
+
+  // Create azure blob client
+  var blob = new azure.Blob({
+    accountId: process.env.AZURE_STORAGE_ACCOUNT,
+    accessKey: process.env.AZURE_STORAGE_ACCESS_KEY
+  });
+
+  suite("Container", function() {
+    test('create container with metadata', function() {
+      var containerName = 'fast-azure-blob-container-with-metadata';
+      var metadata = {
+        testKey: 'testValue'
+      };
+
+      return blob.createContainer(containerName, metadata, null)
+        .then(function(){
+          return blob.deleteContainer(containerName);
+        });
+    });
+
+    test('create container without metadata', function() {
+      var containerName = 'fast-azure-blob-container';
+
+      return blob.createContainer(containerName, {}, null)
+        .then(function(){
+          return blob.deleteContainer(containerName);
+        });
+    });
+
+    test('create container with container access', function() {
+      var containerName = 'fast-azure-blob-container-with-access';
+
+      // TODO verify the listing!!
+      return blob.createContainer(containerName, {}, null)
```

Comment:
You are right! I forgot to add the access level! After I started the test, I realized that if I want to verify all the possible access levels I need the 'List Blobs' and 'List containers' REST endpoints to be implemented. And because of this, I left it as in progress.